typescript/src/workflows/agent.ts

```diff
@@ -59,10 +70,13 @@ export class AgentWorkflow {
     });
   }
 
-  run(messages: Message[], options: WorkflowRunOptions<string> = {}) {
+  run(
+    inputs: (z.infer<typeof AgentWorkflow.schema.shape.inputs.element> | Message)[],
```

Comment:
1. remove the static schema property and make it part of the module
2. here just use `Array<ToolCallingAgentRunInput | Message>`